app/controllers/summary_controller.rb

```diff
@@ -12,12 +12,12 @@ def new
   end
 
   def create
-    @summary = Summary.new(params.require(:summary).permit(:body))
+    @summary = Summary.new(params.require(:summary).permit(:body, :post_id))
```

Comment:
This allows the post_id to be read from the params, but it's going to look inside a `summary` json object, and so I needed to place the id in the form in a hidden field.